src/main/java/com/project/house/rental/dto/PropertyDto.java

```diff
@@ -0,0 +1,41 @@
+package com.project.house.rental.dto;
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class PropertyDto {
+
+    long id;
+
+    String title;
+
```

Comment:
Chưa có validate cho Dto, cần phải valid những trường not null trong database không được trống.
Giá và diện tích phải lớn hơn 0. Có thể cần check status có phải đúng dạng enum trong database hay không.